pyproject.toml

```diff
@@ -23,7 +23,11 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry.dependencies]
 python = ">=3.9,<4.0"
-
+pandas = "^1.5.3"
+pyarrow = "16.1.0"
+numpy = "^1.21.0"
+grpcio = "^1.48.1"
+grpcio-status = "^1.64.1"
```

Comment:
Because these dependencies are optional for PySpark Classic, I would like to have them as [extras](https://python-poetry.org/docs/pyproject/#extras).

Reply:
Addressed it in [this commit](https://github.com/MrPowers/quinn/pull/244/commits/1d8a0de43bfa76e6db740c6ece06d04ed3adc6b2) and renamed it in a [different commit](https://github.com/MrPowers/quinn/pull/244/commits/b44a025b523794dc7c889e187b8b00aa18c352e5)